slides.tex

```diff
@@ -221,7 +254,7 @@
      \end{frame}
 
     \begin{frame}
-        \frametitle{Useful links}
+        \frametitle{More Information}
```

Comment:
Let's think some about what urls we want to include on this page. How about replacing 'PyUmbral' with 'Umbral reference implementation' (not sure about this). Remove goUmbral (since we'll consistently be adding more languages I don't know if it's makes sense to list them all. Add nuFHE.  Add staking economics paper? (or perhaps link to it elsewhere in the presentation)

Reply:
Great. IMO replace umbral-doc with nucypher repo since we almost have that ready for use.